browser/ui/webui/speedreader/speedreader_toolbar_data_handler_impl.cc

```diff
@@ -137,7 +137,7 @@ void SpeedreaderToolbarDataHandlerImpl::AiChat() {
     return;
   }
 #if BUILDFLAG(ENABLE_AI_CHAT)
-  static_cast<BraveBrowserWindow*>(browser_->window())->OpenAiChatPanel();
+  static_cast<BraveBrowserWindow*>(browser_->window())->ToggleAiChatPanel();
```

Comment:
I think we don't need to plumbing to view layer.
We can use `SidePanelUI::GetSidePanelUIForBrowser()->Show()` and `->GetCurrentEntryId()` for toggling.

Reply:
Ah.. maybe this was implemented initially when `SidePanelUI` was in view layer.

Reply:
Thanks, I'll remove the view layer